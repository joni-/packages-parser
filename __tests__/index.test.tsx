import React from "react";
import renderer from "react-test-renderer";
import Home from "../pages/index";

describe("Home", () => {
  it("renders page without changes", () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
