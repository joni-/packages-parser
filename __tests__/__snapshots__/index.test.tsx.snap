// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders page without changes 1`] = `
<h1>
  Display packages list here
</h1>
`;
